VsixExtension.targets

```diff
@@ -0,0 +1,24 @@
+﻿<Project>
+	<!-- Install the dependencies for the LSP server. -->
+	<Target Name="PackAngularServer" BeforeTargets="PreBuildEvent">
+		<Message Text="Installing server dependencies..." />
+		<Exec Command="npm ci" />
+	</Target>
+	<ItemGroup>
+		<!-- TODO: Not sure if this is okay. -->
+		<!-- We exclude the TypeScript localization folder because otherwise the extension path
+		is too long and deployment fails. -->
+		<Content Include="node_modules\**\*.*" Exclude="node_modules\typescript\loc\**\*.*">
```

Comment:
Does this happen for the local (experimental) or actual path? The experimental path is usually longer than the default path that would probably be used by customers. See https://devdiv.visualstudio.com/DevDiv/_git/VS-NodeJs?path=/readme.md&_a=preview&anchor=2.-npm